app/mattermost.js

```diff
@@ -6,14 +6,7 @@ import Orientation from 'react-native-orientation';
 import {Provider} from 'react-redux';
 import {Navigation, NativeEventsReceiver} from 'react-native-navigation';
 import {IntlProvider} from 'react-intl';
-import {
-    Alert,
-    AppState,
-    InteractionManager,
-    Keyboard,
-    NativeModules,
-    Platform
-} from 'react-native';
+import {Alert, AppState, InteractionManager, Keyboard, NativeModules, Platform} from 'react-native';
```

Comment:
You don't need to change these back now, but can we try to have these spread across multiple lines when we're importing this many things? It makes it easier to read and see changes